src/main/scala/org/broadinstitute/dsde/agora/server/AgoraConfig.scala

```diff
@@ -90,4 +92,26 @@ object AgoraConfig {
       case _ => baseUrl
     }
   }
+
+  implicit class EnhancedScalaConfig(val config: Config) extends AnyVal {
```

Comment:
I suspect it was lifted from a code base from the old picard/zamboni group actually :)